scripts/globals/items/beryllium_bolt_quiver.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- ID: 6428
+-- Beryllium Bolt Quiver
+-- When used, you will obtain one stack of Beryllium Bolts
+-----------------------------------
+require("scripts/globals/items")
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(22285, 99)
```

Comment:
Adding to the comment, you added the wrong items to the table.
In this line the item id is 22285 (Wich are beryllium bolts) with 99 being the quantity.
So in the items.lua file, you should have added those. For example...
`BERYLLIUM_BOLT = 22285,`
And then this line should be changed to...
`target:addItem(xi.items.BERYLLIUM_BOLT, 99)`